test/e2e/utils.go

```diff
@@ -259,9 +259,9 @@ func undeployJaegerInstance(jaeger *v1.Jaeger) bool {
 	return true
 }
 
-func getJaegerInstance(name, namespace string) *v1.Jaeger {
+func getJaegerInstance(name, jaegerInstanceNamespace string) *v1.Jaeger {
```

Comment:
Not sure this name change is necessary? The function is saying `getJaegerInstance` so the `name` and `namespace` seem to be implied they are for the jaeger instance?